app/templates/image-set/cv-results.hbs

```diff
@@ -0,0 +1,44 @@
+<h3>CV Results</h3>
+
+<div class='row'>
+  <div class='col-sm-4'>
+    <ul class='list-group'>
+      {{#each model as |cvResult|}}
+        {{lg-cv-result-summary cvResult=cvResult action='setActiveCvResult'}}
```

Comment:
I'm not convinced there needs to be a UI for showing multiple CV results. When I went over this with Justin we had only talked about showing the most recent result set. Can you be in charge of making sure this interface matches Justin's expectations? As long as he wants to see multiple results, this is good by me.